week-3/implement/is-valid-triangle.js

```diff
@@ -38,3 +38,17 @@
 // Then it should return true because the input forms a valid triangle.
 
 // This specification outlines the behavior of the isValidTriangle function for different input scenarios, ensuring it properly checks for invalid side lengths and whether they form a valid triangle according to the Triangle Inequality Theorem.
+
+function isValidTriangle(a,b,c){
+    if (a+b<=c || a+c<=b || b+c<=a){
+        return false;
+    }
+    else if (a<=0 || b<=0 || c<=0){
+        return false;
+    }
+    else if(a+b>c && a+c>b && b+c>a){
```

Comment:
This logic here is the opposite of the logic on line 43. For example, `a + b <= c` must be `false` for us to reach this line of code. Therefore `a + b > c` will always be true at line 49. The same goes for the other statements.

For that reason the checks here on line 49 aren't necessary